pmm-app/tests/QAN/steps/qanActions.js

```diff
@@ -278,14 +278,14 @@ module.exports = {
 
   async verifyPagesAndCount(itemsPerPage) {
     const count = await this.getCountOfItems();
-    const lastpage = await this.getPagesCount();
+    const lastpage = await this.getPageCount();
     const result = count / lastpage;
 
     assert.equal(Math.ceil(result / 25) * 25, itemsPerPage, 'Pages do not match with total count');
   },
 
-  async getPagesCount() {
-    const pagesCount =
+  async getPageCount() {
+    const pageCount =
```

Comment:
Now the names are different on this and the next line. Does it work?

Reply:
Sure you are right!

Reply:
Fixed